msg/analog_measurement.msg

```diff
@@ -0,0 +1,10 @@
+uint64 timestamp				# time since system start (microseconds)
+int32 id						# Source node ID
```

Comment:
Change to: `uint32 device_id` to be consistent with other sensor/driver-related topics